### ISSUE TITLE
AWS SSO Login Failed: Service returned error code AccessDeniedException

### ISSUE THREAD

**Issue by RogerThat000** [aws/aws-toolkit-jetbrains#4130]
- OS: Windows10 
- JetBrains product:IntelliJ
- JetBrains product version: IntelliJ IDEA 2022.1 
- AWS Toolkit version:1.67-221
- AWS CLI version:2.7.33
- JVM/Python version:3.9.11

I have admin access to my accounts, and while trying to use the toolkit to start Code Whisperer, I enter the SSO start URL - enter and copy the code and grant allow access but receive the below message 

Service returned error code AccessDeniedException (Service: SsoOidc, Status Code: 400, Request ID: 3777729b-a103-4743-bb04-e04df62bb9e0)

AWS ToolKit -  has aws credentials set to my Admin profile
[XXXaccount_AWSAdministratorAccess]
aws_access_key_id=XXXX
aws_secret_access_key=XXX
aws_session_token=XXX

sso URL - https://XXXX.awsapps.com/start#
sso region - us-east-1

We are trying to get code whisperer and Q working on one of our sandbox accounts.

**Comment by anikoiau** [comment]
Did you find a solution?

**Comment by RogerThat000** [comment]
Yes I did - I had to go back to my Org Account and re-add all the users in CodeWhisperer profile and then on sandbox account I re tried the steps and it worked.
